process_report/institute_list.yaml

```diff
@@ -0,0 +1,74 @@
+-   display_name: Northeastern University
+    domains:
+    - northeastern.edu
+-   display_name: Boston University
+    domains:
+    - bu.edu
+    - robbaron
```

Comment:
Since we've fully transitioned to using email addresses rather than username (and also have a aliasin system in place), I feel it's safe to remove everything that is an username from this file.